install_deps.sh

```diff
@@ -10,16 +10,39 @@ set -e
 cd "$(dirname "${0}")"
 
 ### Parse args ###
-
+show_help=0
+exit_code=
 assume_yes=
+JOBS=1
```

Comment:
Why not set `illixr_nproc` directly? People used to say about Apple, `JOBS` was superfluous anyway, right xD